docs/content/documentation/content/shortcodes.md

```diff
@@ -130,6 +131,29 @@ A quote
 The body of the shortcode will be automatically passed down to the rendering context as the `body` variable and needs
 to be on a new line.
 
+### Shortcodes with no arguments
```

Comment:
Is that necessary with the paragraph change above?

Reply:
Having an example of an arguments-less shortcode instantiation will help people who are skimming the page. 

I personally tried copying the body example, deleted the arguments, and was wondering why it wasn't working; I figure that there will be others like me.